crates/biome_js_analyze/src/lint/nursery/use_explicit_function_return_type.rs

```diff
@@ -128,6 +130,14 @@ impl Rule for UseExplicitFunctionReturnType {
     type Options = ();
 
     fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let source_type = ctx.source_type::<JsFileSource>().language();
+        let is_ts_source = source_type.is_typescript();
+        let is_declaration = source_type.is_definition_file();
+
+        if is_declaration || !is_ts_source {
+            return None;
+        }
+
```

Comment:
Is there any way to add test cases for this?

Reply:
I suppose just creating a javascript and/or `.d.ts` file inside `useExplicitFunctionReturnType` folder should be enough. If there aren't diagnostics, it should be fine

Reply:
I’ve added the test!!